test/relayclient/KnownRelaysManager.test.ts

```diff
@@ -76,12 +76,15 @@ contract('KnownRelaysManager', function (
       penalizer = await Penalizer.new()
       relayHub = await deployHub(stakeManager.address, penalizer.address)
       config = configureGSN({
-        logLevel: 'error',
-        relayHubAddress: relayHub.address,
+        loggerConfiguration: { logLevel: 'error' },
         relayLookupWindowBlocks
       })
-      logger = createClientLogger(config.logLevel, config.loggerUrl, config.loggerApplicationId, config.loggerUserIdOverride)
-      contractInteractor = new ContractInteractor(web3.currentProvider as HttpProvider, logger, config)
+      logger = createClientLogger(config.loggerConfiguration)
+      contractInteractor = new ContractInteractor({
+        provider: web3.currentProvider as HttpProvider,
```

Comment:
nitpic: I think ContractInteractor should be fixed, so that it accept web3.currentProvider without casting (after all, that's one of its main usages)

Reply:
web3.currentProvider is defined as this:
export type provider =
    | HttpProvider
    | IpcProvider
    | WebsocketProvider
    | string
    | null;

I have no idea where did it make sense for Web3 to allow both 'string' and 'null' as a valid provider type, but I don't want to allow these in the C.I. constructor.